e2e-test/features/DesignTimeSLT.feature

```diff
@@ -0,0 +1,65 @@
+# Copyright © 2021 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+Feature: Design Time SLT Scenario
+
+  @SLT @Design-Time @Sanity
+  Scenario Outline: User is able to provides input to SLT parameters
```

Comment:
->Verify user is able to provides input to SLT parameters

Reply:
Similar comment for all scenario descriptions

Reply:
Working in progress.